other/materials_designer/passivate_nanoribbon.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>in the image below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>